src/components/molecules/ChatMessageBox/index.ts

```diff
@@ -0,0 +1 @@
+export { ChatMessageBox as default } from "./ChatMessageBox";
```

Comment:
We use named exports rather than `default` exports:

```suggestion
export { ChatMessageBox } from "./ChatMessageBox";
```